src/detectors/ScanningStrategyDetector.ts

```diff
@@ -42,6 +43,10 @@ export class ScanningStrategyDetector implements IDetector<string, ScanningStrat
       accessType = await this.determineRemoteAccessType({ remoteUrl: path, serviceType });
     }
 
+    if (accessType === AccessType.private) {
```

Comment:
We should just throw an error and `try/catch` the error in the `index.ts` where should be the CLI logic.

Reply:
What do you mean? Don't ask the user for AT? As you wrote in trello, I thought, that you want it: 
`If the scanner is run without auth credentials and the repository is private, than the CLI asks for credentials with a hint containing URL navigating directly to "generate api token" page.`